guides/common/modules/proc_installing-the-proxmox-plug-in.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Installing_the_Proxmox_Plug-in_{context}"]
+= Installing the Proxmox Plug-in
+
+Install the Proxmox plug-in to attach an Proxmox compute resource provider to {Project}.
+This allows you to manage and deploy hosts to Proxmox.
+
+.Procedure
+. Install the Proxmox plug-in on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# dnf install rubygem-fog-proxmox rubygem-foreman_fog_proxmox
```

Comment:
You don't need to specify `rubygem-fog-proxmox`, since the plugin itself already depends on it. Then it should also use the `{package-install}` macro:
```suggestion
# {package-install} rubygem-foreman_fog_proxmox
```

Though I'd really like to avoid this and instead rely on the installer.

Reply:
I've applied this change using "package-install-project". I will also have a look at the link below on how to get Proxmox added to foreman-installer.